tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -222,7 +269,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 options: {
                     moduleSource: {
                         func: "fluid.tests.tooltipModuleSource",
-                        args: "{pager}"
+                        args: ["{pager}", [2, 3, 4]]
```

Comment:
Please pull out the array into a public variable that is named appropriately so that it is clear what it is.